src/Authzed.Net/Authzed/Api/Materialize/V0/Watchpermissions.cs

```diff
@@ -253,7 +254,11 @@ public void MergeFrom(pb::CodedInputStream input) {
     #else
       uint tag;
       while ((tag = input.ReadTag()) != 0) {
-        switch(tag) {
+      if ((tag & 7) == 4) {
+        // Abort on any end group tag.
+        return;
+      }
+      switch(tag) {
```

Comment:
These changes are present in most files; I imagine this is due to a new protobuf version rather than anything that buf is doing. Note that these files don't contain an annotation of the version of protobuf used to generate them, unlike other languages.